tools/test-tools/package.json

```
@@ -40,5 +40,6 @@
 		"prettier": "~2.6.2",
 		"rimraf": "^2.6.2",
 		"typescript": "~4.5.5"
-	}
+	},
+	"packageManager": "pnpm@7.30.5+sha1.ba4f8057ab7b24db2ebe553808b95adb55b5330f"
```

Does this work? I couldn't find any examples using sha512 with packageManager, but maybe this format works?

```suggestion
	"packageManager": "pnpm@7.30.5+sha512-kXiH7+iGhDcm3UVhjb4pzbRYljoT2KVR8WFL37b+c15F9CuaLau0RTozrXx/9snf1JEmGICjRnMM2XArmM01sg=="
```

---

That form doesn't work unfortunately. I downloaded the tarball manually and used `sha512sum` which produced `917887efe886843726dd45618dbe29cdb458963a13d8a551f1614bdfb6fe735e45f42b9a2dabb4453a33ad7c7ff6c9dfd491261880a346730cd9702b98cd35b2`.

Trying that now.

---

Worked!